modules/ml/src/main/java/org/apache/ignite/ml/clustering/KMeansModel.java

```diff
@@ -18,13 +18,16 @@
 package org.apache.ignite.ml.clustering;
 
 import java.util.Arrays;
+import org.apache.ignite.ml.Exportable;
+import org.apache.ignite.ml.Exporter;
+import org.apache.ignite.ml.KMeansModelFormat;
 import org.apache.ignite.ml.math.DistanceMeasure;
 import org.apache.ignite.ml.math.Vector;
 
 /**
  * This class encapsulates result of clusterization.
  */
-public class KMeansModel implements ClusterizationModel<Vector, Integer> {
+public class KMeansModel implements ClusterizationModel<Vector, Integer>, Exportable<KMeansModelFormat> {
     /** Centers of clusters. */
     private Vector[] centers;
```

Comment:
consider either making centers and distance final, as suggested by code inspection or adding a comment explaining why it's better to keep these non-final

Reply:
added final modifier